specification/paths/ConsolidateShipments.json

```
@@ -0,0 +1,56 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "experimental"
+        ]
+      }
+    ],
+    "summary": "Update given shipments with the provided consolidation shipment id.",
+    "description": "This endpoint updated provided shipment ids with the consolidation shipment id.",
```

Since the summary will be the title on the left, let's keep it short. More information is in the description.
```suggestion
    "summary": "Consolidate shipments.",
    "description": "This endpoint links the provided shipments to a consolidation shipment.",
```